src/components/Carousel.tsx

```diff
@@ -35,112 +38,97 @@ export default function Carousel({
 			})
 		)
 	);
-	const startingIndex = () => {
-		if ((carouselItems.length & 2) === 0) {
-			return Math.ceil(carouselItems.length / 2);
-		} else {
-			return Math.ceil(carouselItems.length / 2);
-		}
```

Comment:
Removed the `if` here as it was setting the same value, and updated it to only select the middle item if there are more than `2` items.